src/app/components/crypto-assets/stacks/fungible-token-asset/stacks-fungible-token-asset-item.layout.tsx

```diff
@@ -37,48 +38,40 @@ export const StacksFungibleTokenAssetItemLayout = forwardRefWithAs(
       subBalance?.amount.isGreaterThan(0) && !balance.amount.isEqualTo(subBalance.amount);
 
     return (
-      <Box
-        as={isPressable ? 'button' : 'div'}
-        display="flex"
-        textAlign="left"
-        outline={0}
-        position="relative"
-        ref={ref}
-        flexGrow={1}
-        spacing="base"
-        {...rest}
-        {...bind}
-      >
-        <Stack alignItems="center" flexGrow={1} isInline spacing="base">
-          <StacksAssetAvatar
-            color="white"
-            gradientString={avatar}
-            imageCanonicalUri={imageCanonicalUri}
-            isUnanchored={isUnanchored}
-            size="36px"
-          >
-            {title[0]}
-          </StacksAssetAvatar>
-          <Stack flexGrow={1} spacing="none">
-            <SpaceBetween width="100%">
-              <Text>{title}</Text>
-              <Tooltip
-                label={formattedBalance.isAbbreviated ? amount : undefined}
-                placement="left-start"
-              >
-                <Text data-testid={title} fontVariantNumeric="tabular-nums" textAlign="right">
-                  {formattedBalance.value}
-                </Text>
-              </Tooltip>
-            </SpaceBetween>
-            <SpaceBetween height="1.25rem" width="100%">
-              <AssetCaption caption={caption} isUnanchored={isUnanchored} />
-              {isUnanchored && subBalance ? <SubBalance balance={subBalance} /> : null}
-            </SpaceBetween>
-          </Stack>
-        </Stack>
-        {component}
-      </Box>
+      <Flex as={isPressable ? 'button' : 'div'} outline={0} ref={ref} {...rest} {...bind}>
+        <Flag
```

Comment:
Just fixing here bc alignment in balances list was off...

<img width="234" alt="Screen Shot 2023-03-14 at 7 38 13 PM" src="https://user-images.githubusercontent.com/6493321/225408085-850803a9-19d2-4b9a-8962-e64f92e4d0cd.png">